test/new_relic/control/security_interface_test.rb

```diff
@@ -13,7 +13,15 @@ def setup
     end
   end
 
+  # For testing purposes, clear out the supportability metrics that have already been recorded.
```

Comment:
This comment doesn't seem to cover the need for the shutdown.

Reply:
Removed.